web/src/features/charts/bar-breakdown/ProductionSourceLegend.tsx

```diff
@@ -9,16 +9,18 @@ export default function ProductionSourceLegend({
   electricityType: ElectricityModeType;
 }) {
   return (
-    <g className="pointer-events-none">
-      <rect
-        fill={modeColor[electricityType as ElectricityModeType]}
-        width={14}
-        height={14}
-        rx={2}
-      />
-      <g transform={`translate(3, 3)`} width={14} height={8}>
-        <ProductionSourceIcon source={electricityType} />
+    <svg width={14} height={14}>
```

Comment:
Could we make them a bit bigger? On my non-retina screen they become really hard to discern :)
![Screenshot 2024-04-18 at 11 19 07](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/8cfd6581-6840-40dd-be74-42653b314ad2)

I guess it's the same in the bar-breakdown-chart at the top, think @Alportan discussed it a some point, but can't remember what the conclusion was 🤔

Reply:
Good point! Yes, I do have another size for the icons. Maybe we can update it in the main component instead so it will reflect on all uses 🙌 

The new size is 16x16 instead, maybe we can try that and see if it helps in this case! 😊

Reply:
I have changed them to be 16x16 now. What do you think?

<img width="449" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/95029552/b468c62f-7d43-4c66-99ec-e906c06fd152">

Reply:
Looking good! I think it's already better 🙌 

@madsnedergaard what are your thoughts?